app/src/main/java/org/sopt/official/domain/entity/attendance/AttendanceInfo.kt

```diff
@@ -0,0 +1,111 @@
+package org.sopt.official.domain.entity.attendance
+
+import java.time.LocalDateTime
+
+data class AttendanceInfo(
```

Comment:
```suggestion
data class Attendance(
```

로 줄이는건 어떨까요? Info는 불필요한 단어인 것 같아서요!

Reply:
ㅇㅈ합니다. 감사합니다!

Reply:
이름 변경했습니다! 839f758758d48c582e2131b128383f657b22fb21
추가적으로 SessionInfo도 Session으로 변경했습니다!